zetaclient/chains/bitcoin/observer/observer.go

```diff
@@ -679,6 +676,7 @@ func (ob *Observer) BuildBroadcastedTxMap() error {
 	return nil
 }
 
+// LoadLastBlock loads last scanned block from DB
```

Comment:
Maybe we can rename `LoadLastScannedBlock`?

Reply:
https://github.com/zeta-chain/node/pull/2222/commits/09acf94869bdd3ea5d92504b90958e38e928b468